TeamCode/src/main/java/org/firstinspires/ftc/teamcode/constants.java

```diff
@@ -0,0 +1,8 @@
+package org.firstinspires.ftc.teamcode;
+
+public interface constants {
+    double clawOpenValue = 1;
```

Comment:
I would call these variables openClawPosition or openClawValue. Something along those lines as it looks better and is a bit more readable